terraform/variables.tf

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+variable "model_name" {
+  description = "Name of Juju model to deploy application to"
+  type        = string
+  default     = ""
+}
+
+variable "app_name" {
+  description = "Name of the application in the Juju model"
+  type        = string
+  default     = "mongodb"
+}
+
+variable "channel" {
+  description = "The channel to use when deploying a charm"
+  type        = string
+  default     = "6/beta"
+}
+
+variable "mongo-config" {
+  description = "Additional configuration for the MongoDB"
+  default     = {}
+}
```

Comment:
```suggestion
variable "config" {
  description = "Additional configuration for the MongoDB. Details about available options can be found at https://charmhub.io/mongodb-k8s/configure?channel=6/beta."
  type        = map(string)
  default     = {}
}
}
```

Reply:
I think using only config will be better as there is not other config parameter here.